pkg/generate/command.go

```diff
@@ -0,0 +1,35 @@
+package generate
+
+import (
+	"fmt"
+	"log"
+	"github.com/budimanjojo/talhelper/pkg/config"
+	"github.com/budimanjojo/talhelper/pkg/talos"
+)
+
+func GenerateCommand(c *config.TalhelperConfig, outDir string, generateForNode string, generateApply bool, generateUpgrade bool) error {
+	if !generateApply && !generateUpgrade {
+		fmt.Printf("Must select one of `--apply` of `--upgrade`\n")
+		return nil
+	}
+
+	for _, node := range c.Nodes {
+		var isSelectedNode = generateForNode != "" && generateForNode == node.IPAddress
+		if generateForNode == "" || isSelectedNode {
+			if generateApply {
+				fileName := outDir + "/" + c.ClusterName + "-" + node.Hostname + ".yaml"
+				fmt.Printf("talosctl apply-config --talosconfig %s/talosconfig --nodes %s --file %s --insecure;\n", outDir, node.IPAddress, fileName)
+			}
+			
+			if generateUpgrade {
+				var image, err = talos.GetInstallerURL(node.Schematic, "factory.talos.dev", c.TalosVersion)
```

Comment:
I think this will give unexpected result when `node.Schematic` is `nil`. Also `"factory.talos.dev"` shouldn't be hardcoded, maybe another flag?
And instead of `c.TalosVersion`, use `c.GetTalosVersion()` instead so it will handle in case user doesn't define `talosVersion` in the config file.